README.md

```diff
@@ -33,4 +33,15 @@ This project is perfect for people starting out with opensource and knows basics
 <img src="https://github.com/mudit023/templates/blob/main/theme-toggle/Screenshot-Mobile.png" width=400px height=518px align="left">
 <img src="https://github.com/mudit023/templates/blob/main/background-slider/Screenshot-Mobile.png" width=400px>
 
-### Want to contribute to the project? Read the [Contribution Guidelines](https://github.com/mudit023/templates/blob/main/CONTRIBUTING.md), [Template Guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md), and adhere to our [Code of Conduct](https://github.com/mudit023/templates/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
Again you deleted the change I just made.
See
![Screenshot 2022-04-16 012252](https://user-images.githubusercontent.com/71916646/163625948-f42684a9-ddad-46cf-9430-7382c1e3bbe3.png)
**Note:** Always fetch upstream and keep your fork up to date and then only create a PR

Reply:
Ok